CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.2.1 - May 15, 2020
+### Improvements and bug fixes
+ - Fixed a crash when `Layer.getFilter()` has expression `distance` content. ([#380](https://github.com/mapbox/mapbox-gl-native-android/pull/380))
+ - Fixed a crash when rotation on an Android 4.2.2 x86 tablet. ([#381](https://github.com/mapbox/mapbox-gl-native-android/pull/381)（h/t [ystsoi](https://github.com/ystsoi))
```

Comment:
This looks good, and thanks for remembering the h/t! but I have a slight preference for going with the more specific changelog entry in @tobrun's PR, which seems to be the right level of detail:

> Reset AndroidRendererBackend properly to prevent a crash during rotation on an Android 4.2.2 x86 tablet.